python/servo/command_base.py

```diff
@@ -1037,7 +1037,7 @@ def ensure_rustup_version(self):
             raise
         version = tuple(map(int, re.match(b"rustup (\d+)\.(\d+)\.(\d+)", version_line).groups()))
         if version < (1, 21, 0):
-            print("rustup is at version %s.%s.%s, Servo requires 1.11.0 or more recent." % version)
+            print("rustup is at version %s.%s.%s, Servo requires 1.21.0 or more recent." % version)
```

Comment:
Let's make this a variable that's used in the check as well to avoid this happening again in the future.